datafusion/core/src/datasource/file_format/csv.rs

```diff
@@ -770,7 +770,7 @@ mod tests {
                 "c7: Int64",
                 "c8: Int64",
                 "c9: Int64",
-                "c10: Int64",
+                "c10: Utf8",
```

Comment:
Changed due to arrow changes: https://github.com/apache/datafusion/pull/12724/files#r1785141290